builtin/mainmenu/tab_credits.lua

```diff
@@ -19,61 +19,68 @@
 
 local core_developers = {
 	"Perttu Ahola (celeron55) <celeron55@gmail.com>",
-	"Ryan Kwolek (kwolekr) <kwolekr@minetest.net>",
 	"sfan5 <sfan5@live.de>",
-	"kahrl <kahrl@gmx.net>",
-	"sapier",
 	"ShadowNinja <shadowninja@minetest.net>",
 	"Nathanaël Courant (Nore/Ekdohibs) <nore@mesecons.net>",
 	"Loic Blot (nerzhul/nrz) <loic.blot@unix-experience.fr>",
-	"Matt Gregory (paramat)",
-	"est31 <MTest31@outlook.com>",
+	"paramat",
 	"Craig Robbins (Zeno) <craig.d.robbins@gmail.com>",
 	"Auke Kok (sofar) <sofar@foo-projects.org>",
-	"Andrew Ward (rubenwardy) <rubenwardy@gmail.com>",
+	"rubenwardy <rw@rubenwardy.com>",
+	"Krock/SmallJoker <mk939@ymail.com>",
 }
 
 local active_contributors = {
-	"Duane Robertson <duane@duanerobertson.com>",
-	"SmallJoker <mk939@ymail.com>",
-	"Lars Hofhansl <larsh@apache.org>",
-	"Jeija <jeija@mesecons.net>",
-	"Gregory Currie (gregorycu)",
-	"Sokomine <wegwerf@anarres.dyndns.org>",
-	"TeTpaAka",
-	"Jean-Patrick G (kilbith) <jeanpatrick.guerrero@gmail.com>",
-	"Diego Martínez (kaeza) <kaeza@users.sf.net>",
-	"Dániel Juhász (juhdanad) <juhdanad@gmail.com>",
-	"Rogier <rogier777@gmail.com>",
+	"red-001 <red-001@outlook.ie> [CSM & Menu fixes]",
+	"Dániel Juhász (juhdanad) <juhdanad@gmail.com> [Audiovisuals: lighting]",
+	"numberZero [Audiovisuals: meshgen]",
+	"Lars Hofhansl <larsh@apache.org> [Occulusion culling, fixes]",
+	"Jean-Patrick G (kilbith) <jeanpatrick.guerrero@gmail.com> [Audiovisuals]",
+	"Vincent Glize (Dumbeldor) <vincent.glize@live.fr> [CSM]",
+	"bigfoot547 [CDM]",
```

Comment:
Is CDM a thing? Or is this a typo?
By the way thank you so much for adding me! :smile: